sops.go

```diff
@@ -105,10 +108,10 @@ type Tree struct {
 
 // TrimTreePathComponent trimps a tree path component so that it's a valid tree key
 func TrimTreePathComponent(component string) (string, error) {
-	if component[len(component)-1] != ']' {
+	if component[len(component) - 1] != ']' {
 		return "", fmt.Errorf("Invalid component")
 	}
-	component = component[:len(component)-1]
+	component = component[:len(component) - 1]
```

Comment:
No, but I switched from gofmt to goimports, and it seems they differ in a few small things. I don't know whether it was you or oremj that told me to use goimports because it sorts imports 😜